Taho/ctahoopt.cpp

```diff
@@ -23,9 +23,7 @@ CTahoOpt::CTahoOpt(QWidget *parent) :
     ui(new Ui::CTahoOpt)
 {
     ui->setupUi(this);
-//DYJ Taho 4.07j     m_pDefPrg=ui->rb_pfadPrg;
-//DYJ Taho 4.07j     m_pDefED=ui->rb_pfadED;
-    m_tasks = QThread::idealThreadCount();
+    m_tasks = abs(QThread::idealThreadCount());
```

Comment:
abs() will not fix signed/unsigned clash, it is int -> int function
```suggestion
    m_tasks = static_cast<unsigned>(QThread::idealThreadCount());
```